src/components/announcements/announcement.js

```diff
@@ -0,0 +1,109 @@
+                  return (
+                    <div className="announcement">
+                      <div className="topannouncement">
+                        <Avatar
+                          className="avatar"
+                          src={announcement.avatarUrl}
+                        ></Avatar>
+                        <div className="text">
+                          <div>{announcement.description}</div>
+                          <div className="bottomannouncement">
+                            {announcement.author}, {announcement.topicName}
+                          </div>
+                        </div>
+                        <IconContext.Provider
+                          value={{ color: "#F04B58", size: "24px" }}
+                        >
+                          <div className="icon">
+                            <MdInfoOutline />
+                          </div>
+                        </IconContext.Provider>
+                      </div>
+                    </div>
+                  );
+                })}
+              </div>
+            </Tab>
+            <Tab eventKey="Recents" title="Recents" className="recenttab">
+              <div className="announcements">
+                <div className="announcement">
+                  <div className="topannouncement">
+                    <Avatar className="avatar"></Avatar>
+                    <div className="text">
+                      <div>
+                        @Jaskirat Singh Mentioned you in the comments in the
+                        Topic: “Random”.
+                      </div>
+                    </div>
+                  </div>
+                </div>
+                <div className="announcement">
+                  <div className="topannouncement">
+                    <Avatar className="avatar"></Avatar>
+                    <div className="text">
+                      <div>
+                        @Jaskirat Singh Mentioned you in the comments in the
+                        Topic: “Random”.
+                      </div>
+                    </div>
+                  </div>
+                </div>
+                <div className="announcement">
+                  <div className="topannouncement">
+                    <Avatar className="avatar"></Avatar>
+                    <div className="text">
+                      <div>
+                        @Jaskirat Singh Mentioned you in the comments in the
+                        Topic: “Random”.
+                      </div>
+                    </div>
+                  </div>
+                </div>
+                <div className="announcement">
+                  <div className="topannouncement">
+                    <Avatar className="avatar"></Avatar>
+                    <div className="text">
+                      <div>
+                        @Jaskirat Singh Mentioned you in the comments in the
+                        Topic: “Random”.
```

Comment:
=>

Reply:
The announcement functionality is not implemented yet, hence using placeholder for same.

Reply:
Can we create a separate component for a Recent and Top Announcement?

Reply:
Yeah okay it can done, no issues.